synapse/util/__init__.py

```diff
@@ -116,6 +116,11 @@ def looping_call(
 
         Waits `msec` initially before calling `f` for the first time.
 
+        If the function given to `looping_call` returns an awaitable/deferred, the next
```

Comment:
https://docs.twistedmatrix.com/en/stable/api/twisted.internet.task.LoopingCall.html only mentions `deferred` but it looks like the [source](https://github.com/twisted/twisted/blob/be28750c20fa55aa344d608a1c0edbfc6da45066/src/twisted/internet/task.py#L251) uses [`maybeDeferred`](https://docs.twistedmatrix.com/en/stable/api/twisted.internet.defer.html#maybeDeferred) which works with coroutines/awaitables (which makes this accurate)

> If `f` returns a deferred, rescheduling will not take place until the deferred has fired.